Libraries/Network/__tests__/FormData-test.js

```diff
@@ -55,4 +55,27 @@ describe('FormData', function() {
     };
     expect(formData.getParts()[0]).toMatchObject(expectedPart);
   });
+
+  it('should return only provided key', function() {
+    formData.append('photo', {
+      uri: 'arbitrary/path',
+      type: 'image/jpeg',
+      name: 'photo.jpg',
+    });
+
+    formData.append('video', {
+      uri: 'arbitrary/path',
+      type: 'video/mp4',
+      name: 'video.mp4',
+    });
+
+    expect(formData.getAll('video').length).toBe(1);
+
+    ['image', 'video'].forEach(type => {
+      const parts = formData.getAll(type);
+      parts.forEach(part => {
+        expect(part.fieldName).toBe(type);
```

Comment:
The array returned by [FormData.getAll](https://l.facebook.com/l.php?u=https%3A%2F%2Fdeveloper.mozilla.org%2Fen-US%2Fdocs%2FWeb%2FAPI%2FFormData%2FgetAll&h=AT1DICyBzwY0RFcIL2ytCyiabGZwEJhE7tW--cZnqSv7SbB75QrEnnVX5Ami2RSQVFdZAc0M9I3eqah_FPaajAwA6YaRTLEw0-FDoTIk0EFyjRBCSd7fD9Dloke4hLMdeC_-FAl_Dgpx2mzn3s9v8dRsn_k) should contain the values. It should not be the internal object representation in FormData.

Reply:
Fixed under 0c33daad69a.